modules/ROOT/pages/configure-monitoring-cloudhub.adoc

```diff
@@ -49,8 +49,8 @@ Anypoint Monitoring non-AM images receive patch updates as soon as they are avai
 | Metrics and logs
 
 | Anypoint Platform (CloudHub)
```

Comment:
We support Metrics and Logs in any 3.8.7, 3.9.x, 4.x.x runtime. Should we condense versions above accordingly?

Reply:
I think that would be best.